server/store/store.go

```diff
@@ -200,21 +210,6 @@ func DeleteConnection(instanceID, mattermostUserID string) (returnErr error) {
 		return err
 	}
 
-	// Check for whether the admin token stored for each confluenceURL is of the current user or not. If it is then delete that admin connection also
```

Comment:
Why is this not needed anymore?

Reply:
The security team advised us not to save the connection details of the admin to make API calls for the user event who are not connected to mattermost.